src/renderer/store/modules/config.js

```diff
@@ -4,20 +4,29 @@
  */
 
 const namespaced = true;
+const defaultApiBaseUrl = 'https://papermatter.app/app'; // prod
+// const defaultApiBaseUrl = 'http://127.0.0.1:8000/app'; // local
 
 const state = {
-  apiBaseUrl: 'https://papermatter.app/app', // prod
-  // apiBaseUrl: 'http://127.0.0.1:8000/app', // local
+  apiBaseUrl: defaultApiBaseUrl
 };
 
 const mutations = {
-  SAVE_API_BASE_URL (state, apiBaseUrl) {
-    state.apiBaseUrl = apiBaseUrl;
+  UPDATE_API_BASE_URL (state, apiBaseUrl) {
+    if (apiBaseUrl === ''){
+      state.apiBaseUrl = defaultApiBaseUrl;
+    } else {
+      state.apiBaseUrl = apiBaseUrl.split('#')[0]; // get ride of eventual anchor
```

Comment:
Isn't it better to clean the URL pasted by the user and only keep the host domain? We know the path to the API so this will be more user proof if the user could paste any URL from his/her instance.

Reply:
I thought of it but I was not sure if self-hosted user will be able to install Paper Matter inside a subfolder of their server?  eg. `example.com/pm/app/api`

Reply:
Subfolder install is not tested so I don't think it will work.

Reply:
Maybe use https://nodejs.org/docs/latest-v10.x/api/url.html to parse the URL. It would be more robust.

Reply:
@antoinelibert fixed with f809c89